content/W2017/W17-event-techtalk-with-Monica-Beckwith.md

```diff
@@ -0,0 +1,24 @@
+Title: Tech Talk with Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: social, talks
+Slug: tech-talk-with-Monica-Beckwith
+Author: Fatima Taj
+Summary: Monica Beckwith will be visiting campus to give a talk on her work with the Java Virtual Machine.
+
+Monica Beckwith has worked with the Java Virtual Machine for more than a decade
+not just optimizing the JVM heuristics, but also improving the 
+Just-in-time(JIT) code quality for various processor architectures as well as
+working with the garbage collectors and improving garbage collection for server
+systems.
+
+During this talk, Monica will cover a few JIT and Runtime optimizations and she
+will dive into the HotSpot garbage collection and provide an overview of the
+various garbage collectors available in HotSpot.
+
```

Comment:
add eventbrite please - and change it to look more like the other PR you closed + my comments? :3